CautionaryAlertsApi.Tests/DatabaseTests.cs

```diff
@@ -14,6 +15,8 @@ public class DatabaseTests
         [SetUp]
         public void RunBeforeAnyTests()
         {
+            AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
```

Comment:
This doesn't belong in the PR based on the PR title & current description.
As such, if it's intended to be in here, it would be good to at least mention it in the PR description under "notes". _(So it would be cleat that it's not a left over of some other piece of work that sneaked in without getting noticed)_